python/Ganga/Core/GangaRepository/SessionLock.py

```diff
@@ -484,17 +489,17 @@ def global_lock_setup(self):
                 if not os.path.exists(lock_path):
                     os.makedirs(lock_path)
                 if not os.path.isfile(lock_file):
-                    lock_file_hand = open(lock_file, "w")
-                    lock_file_hand.close()
+                    with open(lock_file, "w"):
+                        pass
```

Comment:
OK, I was just wanting to be certain I think it used to be this way but there was a problem with contexts in the old tests ages ago which required us to open/close a lot more